tests/test_api_v1.py

```diff
@@ -118,6 +119,10 @@ def test_fetch_paginated_requests(mock_chain, app, auth_env, client, db):
             db.session.add(request)
     db.session.commit()
 
+    payload = {'dependencies': sample_deps}
+    client.patch('/api/v1/requests/1', json=payload, environ_base=worker_auth_env)
+    client.patch('/api/v1/requests/11', json=payload, environ_base=worker_auth_env)
```

Comment:
What's the purpose of this?

Reply:
So this is to test the `verbose` flag. Without this, there will always be 0 dependencies in all the requests. I added dependencies to the first request of the first page and the second page. And I add the `verbose` flag while requesting the second page